roles/openshift_logging/tasks/start_cluster.yaml

```diff
@@ -0,0 +1,107 @@
+---
+- shell: >
+    echo "{{openshift_logging_fluentd_nodeselector}}" | cut -d':' -f1
+  register: openshift_logging_fluentd_nodeselector_key
+  check_mode: no
+
+- shell: >
+    echo "{{openshift_logging_fluentd_nodeselector}}" | cut -d' ' -f2
+  register: openshift_logging_fluentd_nodeselector_value
+  check_mode: no
+
+- shell: >
+    {{openshift.common.client_binary}} --config={{mktemp.stdout}}/admin.kubeconfig get node -o name | sed "s,^node/,,g"
```

Comment:
Can jsonpath be used to get the correct output here rather than using shell and sed?